docs/Architecture.md

```diff
@@ -1,28 +1,19 @@
 ![architecture](img/architecture_details.svg)
 
 Nussknacker consists of three parts:
-* engine
-* ui
-* integrations
 
-*Engine* is a **library** which transforms json representation of graph with Nussknacker process into Flink job.
+**engines** - are **libraries** which transforms internal json process representation (process graph) into jobs. For example Flink engine generates Flink specific code, compiles and packages all needed components into JAR file for execution.
```

Comment:
transform

Reply:
for execution, and then runs the job via Flink REST API

Reply:
changed